config/ConfigTemplates.php

```diff
@@ -942,7 +942,17 @@
 		],
 		'whitelistIp2fa' => [
 			'default' => [],
-			'description' => "IP address whitelisting.\nAllow access without 2FA."
+			'description' => "IP address whitelisting.\nAllow access without 2FA.",
+			'validation' => function () {
```

Comment:
```suggestion
			'validation' => '\App\Validator::ip'
```
and one 'return' in function